examples/wind-polar-selections.py

```diff
@@ -0,0 +1,96 @@
+from dash import Dash, html, dcc, callback, Input, Output
+import dash_bootstrap_components as dbc
+import plotly.express as px
+
+app = Dash(__name__, external_stylesheets=[dbc.themes.CYBORG])
+
+df = px.data.wind()
+df[["min_strength", "max_strength"]] = (
+    df.strength.str.replace(r"6\+", "6-7").str.split("-", expand=True).astype(int)
+)
+
+dropdown_1 = dbc.Select(
+    options=[{"label": v, "value": v.lower()} for v in ("Bar", "Line")],
+    value="bar",
+    required=True,
+)
+
+range_1 = dcc.RangeSlider(
+    0,
+    7,
+    step=1,
+    marks={**{v: str(v) for v in range(7)}, 7: ">6"},
+)
+
+graph_1 = dcc.Graph(responsive=True)
+
+blurb = (
+    "The chart illustrates the distribution of wind speeds by direction, "
+    "revealing a significant bias towards winds originating from the west. "
+    "This notable preponderance of westerly winds suggests the influence of "
+    "macro-scale climatic patterns, such as prevailing westerlies common in "
+    "mid-latitude regions. Higher wind speeds are predominantly clustered "
+    "within the western sector, indicating that stronger wind events mostly "
+    "come from this direction. This pattern could be reflective of regional "
+    "topographical influences or larger synoptic systems driving westerly "
+    "flows."
+)
+
+app.layout = dbc.Container(
+    [
+        html.H1("Windy wind things"),
+        html.Div(
+            children=[
+                dbc.Row(
+                    [
+                        dbc.Col(
+                            [
+                                html.P(blurb),
+                                dbc.Label("Filter wind intensity"),
+                                range_1,
+                                dbc.Label("Select chart type"),
+                                dropdown_1,
+                            ],
+                            lg=4,
+                        ),
+                        dbc.Col(graph_1, lg=8),
+                    ]
+                )
+            ],
+        ),
+    ]
+)
+
+
+@callback(
+    Output(graph_1, "figure"), Input(range_1, "value"), Input(dropdown_1, "value")
+)
+def update_figure(strength_range, graph_type):
+    df_copy = df.copy()
+
+    if strength_range:
+        start, stop = min(strength_range), max(strength_range)
+        df_copy = df_copy[(df.min_strength >= start) & (df.max_strength <= stop)]
```

Comment:
This can be simplified to 
```
 dff = df[(df.min_strength >= strength_range[0]) & (df.max_strength <= strength_range[1])]
```

Reply:
I think without adding `value=(0,7)` to the rangeselect, this `strength_range` was `None` on init, which meant the `if` control flow was required.

I also decided to use `min` and `max` because you can cross the points over in the rangeslider, and I thought this might mean `range[0]` was larger than `range[1]` so played it safe.

Have removed the `if` statement but the `start`, `stop` stays for me